CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+<<<<<<< HEAD
 ## Version [3.3.2] - [2019-27-06]
+=======
```

Comment:
Looks like you have an unresolved merge conflict

Reply:
Sorry about it, I forgot to save the file before committing!